README.md

```diff
@@ -8,19 +8,13 @@ which are not available in the official library.
 
 Note: librespot only works with Spotify Premium
 
-# THIS FORK
-As the origin is no longer maintained I wanted to have a place for a version of librespot with other peoples forks and features merged.
+## This fork
+As the origin by [plietar](https://github.com/plietar/) is no longer actively maintained I wanted to have a place for a version of librespot with other peoples forks and features merged.
 
+# Wiki
+More information can befound in the [wiki](https://github.com/librespot-org/librespot/wiki)
```

Comment:
Typo: _befound_

Reply:
Thank you! I knew there would be some thing I missed.